pom.xml

```diff
@@ -63,10 +63,12 @@
         <!-- temporary fix by pinning the version until we upgrade to a version of common that contains this or newer version.
             See https://github.com/confluentinc/common/pull/332 for details -->
         <dependency.check.version>6.1.6</dependency.check.version>
-        <hadoop.version>3.2.4</hadoop.version>
+        <hadoop.version>3.3.6</hadoop.version>
+        <jackson.version>2.15.2</jackson.version>
```

Comment:
jackson dependency is resolved in this PR - 
https://github.com/confluentinc/kafka-connect-storage-cloud/pull/669/files